velox/expression/PrestoCastHooks.cpp

```diff
@@ -32,6 +32,10 @@ PrestoCastHooks::PrestoCastHooks(const core::QueryConfig& config)
   }
 }
 
+bool PrestoCastHooks::canCastIntToBinary() const {
```

Comment:
Isn't Presto's cast to binary behavior as same as Spark's?

Reply:
No, Presto throws exception for the cast from int types to varbinary.